packages/react-tooltip/src/components/Tooltip/index.ts

```diff
@@ -2,4 +2,4 @@ export * from './Tooltip';
 export * from './Tooltip.types';
 export * from './renderTooltip';
 export * from './useTooltip';
-export { useTooltipStyles } from './useTooltipStyles';
+export { tooltipClassName, useTooltipStyles } from './useTooltipStyles';
```

Comment:
Can't you do `export * from './useTooltipStyles';`? It seems that only `tooltipClassName` and `useTooltipStyles` are the only things exported from that file.

Reply:
Sure, done 👍